README.md

```diff
@@ -60,6 +60,51 @@ This option enables you to develop your custom MATLAB commands for running tests
 
   ![new_custom_script_example](https://user-images.githubusercontent.com/47204011/55625021-32904400-57c5-11e9-86b7-478b930796c0.png)
 
+## Configuring "Multi-configuration"(matrix) project.
+Jenkins plugin for MATLAB can be configured to run "Multi-configuration" (matrix) projects. Which allows user to run the same build steps with different configurations, such as testing on multiple environments or multiple MATLAB versions installed on the system.
+
+#### Matrix build for "Automatic" option.
+
+1) Create a "Multi-configuration" project.
+
+![image](https://user-images.githubusercontent.com/47204011/62458632-0e586a00-b79b-11e9-8611-3671adb8c289.png)
+
+2) Add User-defined axis for different MATLAB versions as shown below.
+
+![image](https://user-images.githubusercontent.com/47204011/62459358-c6d2dd80-b79c-11e9-90dc-37891b8179ea.png)
+
+3) Replace the user defined axis variable in MATLAB root appropriately.
+
+![image](https://user-images.githubusercontent.com/47204011/62459137-3c8a7980-b79c-11e9-9bee-305b4cabfd42.png)
+
+4) Save and run the build.
+
+#### Matrix build for "Custom" option.
+
+1) Create a "Multi-configuration" project.
+
+![image](https://user-images.githubusercontent.com/47204011/62458632-0e586a00-b79b-11e9-8611-3671adb8c289.png)
+
+2) Add User-defined axis for different MATLAB versions as shown below.
+
+![image](https://user-images.githubusercontent.com/47204011/62459358-c6d2dd80-b79c-11e9-90dc-37891b8179ea.png)
+
+3) Add another User-defined axis for custom commands as shown below.
```

Comment:
Is it the same Value 'runtests pwd' that you give for all 'Custom' Command option? If not, what other values would be given in there

Reply:
No, I was trying to show how different MATLAB custom commands can be used for each build. However, as Andy's suggestion, I have changed it to a more practical example with TEST_TAG which would replace the values of theseTEST tag against custom command  for eg:  results = runtests(pwd,'TestTag',$TEST_TAG)